invoicer_project/clients/views.py

```diff
@@ -33,17 +32,15 @@ def create(self, request):
     def list(self, request):
         user = get_user_from_jwt(request.headers)
         clients = Client.objects.filter(user_id=user['user_id'])
-        clients_json = ClientSerializer(clients, many=True)
         if len(clients) == 0:
             return JsonResponse(
-                {"first_name": user['first_name'], "last_name": user['last_name'], 'message': 'Not found'}, status=200)
+                {"first_name": user['first_name'], "last_name": user['last_name'], 'message': 'Not found'}, status=404)
```

Comment:
I don't think we should response with message 404. Response with empty list is OK.